pandas/core/window.py

```diff
@@ -472,6 +472,7 @@ class Window(_Window):
     on : string, optional
         For a DataFrame, column on which to calculate
         the rolling window, rather than the index
+    axis : int or string, default 0
```

Comment:
when specifying parameter types: string --> str  (a couple other instances of this elsewhere too)